README.md

```diff
@@ -54,9 +54,10 @@ hep.histplot(h, bins)
 ```
 #### 2D Histograms
 ```
+import numpy as np
 xbins, ybins = [0, 1, 2, 3], [0, 1, 2, 3]
-H = [[2,3,2], [1,2,1], [3,1,3]]
-hep.hist2dplot(H, xbins, ybins)
+H = np.array([[2,3,2], [1,2,1], [3,1,3]])
+hep.hist2dplot(H.T, xbins, ybins)

```

Comment:
The previous line `np.array` is correct but the transpose is not needed.

Reply:
All right, I will remove it.